msgid ""
msgstr ""
"Project-Id-Version: pylav\n"
"POT-Creation-Date: 2023-01-31 23:27+0000\n"
"PO-Revision-Date: 2023-02-01 00:10\n"
"Last-Translator: \n"
"Language-Team: Vietnamese\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.4.2\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Crowdin-Project: pylav\n"
"X-Crowdin-Project-ID: 535244\n"
"X-Crowdin-Language: vi\n"
"X-Crowdin-File: /lib/pylav/extension/bundled_node/locales/messages.pot\n"
"X-Crowdin-File-ID: 3385\n"
"Language: vi_VN\n"

#: pylav/extension/bundled_node/manager.py:227
msgid "You are attempting to run the managed Lavalink node on an unsupported machine architecture"
msgstr "Bạn đang cố gắng chạy nút Lavalink được quản lý trên kiến trúc máy không được hỗ trợ"

#: pylav/extension/bundled_node/manager.py:452
msgid "The output of `{java_exc} -version` was unexpected\n"
"{version_info}"
msgstr "Đầu ra của '{java_exc} -version' thật bất ngờ\n"
"{version_info}"

#: pylav/extension/bundled_node/manager.py:472
msgid "Port {port} already in use. Managed Lavalink startup aborted"
msgstr "Cảng {port} đã được sử dụng. Công ty khởi nghiệp Lavalink được quản lý bị hủy bỏ"

#: pylav/extension/bundled_node/manager.py:477
msgid "Lavalink failed to start: {line}"
msgstr "Lavalink không khởi động được: {line}"

